tests/cpp/test_id_model.cpp

```diff
@@ -2614,4 +2615,33 @@ TEST_F(IdModelTest, ParallelTypePropagation) {
       << "Parallel type propagation failed";
 }
 
+TEST_F(IdModelTest, RepresentativeId) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  auto tv0 = makeConcreteTensor({-1, 1});
+  auto tv1 = makeConcreteTensor({-1, -1});
+  fusion.addInput(tv0);
+  fusion.addInput(tv1);
+
+  auto tv2 = add(tv0, tv1);
+  auto tv3 = sum(tv2, {0, 1});
+  fusion.addOutput(tv3);
+
+  IdModel id_model(&fusion);
+  const ValGraph& graph = id_model.idGraph(IdMappingMode::PERMISSIVE);
```

Comment:
I'm thinking about removing the PERMISSIVE graph. Any particular reason to use it?

Reply:
Right. I actually just wanted a graph with multiple IterTypes in it. I could instead just build a graph manually.

Reply:
Test was changed to build graph directly instead.